src/test/frontend/deploy/deployfromsettings_controller_test.js

```diff
@@ -104,4 +108,53 @@ describe('DeployFromSettings controller', () => {
 
     expect(result).toEqual(expected);
   });
+
+  it('should not send empty description and container commands', () => {
```

Comment:
This test is specifically about empty fields (or, call it, _falsey_). They can be modified, and then erased to empty values.